src/app/components/LogIn/index.tsx

```diff
@@ -57,7 +59,7 @@ export const Login: React.FC<ILogin> = ({ user, logIn, signUp }) => {
     setPassword("");
   };
 
-  // use effct user => redirect
+  useEffect(() =>  user ? history.push("/") : history.push("/auth"), [user, history]);
 
   return (
     <React.Fragment>
```

Comment:
Seems there is no need to use Fragment here. Also try Redirect here like you did it in the Home component.